docs/rfcs/XeGPU.md

```diff
@@ -701,6 +700,332 @@ An example on how to perform transpose using load with chunk_size in SIMT flavor
 
 ```
 
+## Workgroup level XeGPU Operations
+
+By allowing XeGPU operating on workgroup level data size, it provides a concise IR for tensor compiler instead of multiple level nested loop IR for subgroup and work item level operation. To enable XeGPU operate the workgroup level, we introduce `wg_map` attribute to specify how the data is distributed across subgroups. `wg_map` enables tensor compiler to express the cooperative operation among subgroups by specifying a `wg_map` to partition data among subgroups without modifying the IR representation other required when using loop nest IR. The attribute allows tensor compiler to control the block size for both the workgroup and subgroup and perform autotuning as the number of subgroups, layout, and tensor size per subgroups are critical performance knobs.
```

Comment:
suggest using `lane` instead of `wi`.

Reply:
Both lane and work item used in GPU dialect.
The problem of using "lane_layout" is more like describing the layout of hardware. 
"wi_layout" convey the meaning that it is about the layout of wi threads.

Reply:
can you elaborate a bit what is the reason you likes "lane" more than "wi"? @nmostafa  Still struggling on the name.